guardian/manage-frontend
# Upgrade Source to v2.8.2

## What does this change?

Upgrades to Source v2.8.2, making all `@guardian/src-*` package [version numbers consistent](https://theguardian.design/2a1e5182b/p/876251-getting-started/t/9634cf). 

I've made a couple of small changes following the [Source v2 migration guide](https://docs.google.com/document/d/1y2WNKiVifbPRuyTVYYwHCJIgEuFhd4_c8CnKkblCouI/). 

## How to test

The build seems to pass! There should be no visual or functional changes as a result of this change.

## How can we measure success?

Getting all packages up to v2 will make it easier to [upgrade to v3](https://docs.google.com/document/d/1qmHSCCCRs98LI9vN2B8I0yjqBAsRfJclEqcPma0wqbI/). Source v3 requires a bump to Emotion, so is likely to be a larger task.

## Have we considered potential risks?

This is a very low risk upgrade, with no changes to functionality or visuals. TypeScript has done the hard work by identifying breaking changes to the API.

## Images

There are no visual changes.
